pkg/cmd/server/api/helpers.go

```diff
@@ -34,8 +47,8 @@ func (fl *FeatureList) Add(items ...string) error {
 	unknown := []string{}
 	toAppend := make([]string, 0, len(items))
 	for _, item := range items {
-		feature, exists := knownOpenShiftFeatureSet[strings.ToLower(item)]
-		if !exists {
+		feature := NormalizeOpenShiftFeature(item)
```

Comment:
I would keep the return value `(string, bool)`, checking len(...)==0 to determine validity isn't great